lib/facter/rh_certificates.rb

```diff
@@ -0,0 +1,36 @@
+def extract_certificates_from_subscription_manager
+  certificates = {}
+  certificate_end_path = '/cert.pem'
+  private_key_end_path = '/key.pem'
+  data = Facter::Core::Execution.exec('subscription-manager config')
+  data = data.gsub("\n", "")
+  data_array = data.scan(/(\S+)\s*=\s* ([^ ]+)/)
+  data_hash = Hash[data_array]
+  consumer_cert_dir = data_hash["consumercertdir"].gsub(/[\[\]]/, "")
```

Comment:
Right, /etc/rhsm/ca/katello-server-ca.pem is Katello and the non-default value...

[/etc/rhsm/ca/redhat-uep.pem] is the default value, that system wasn't reigstered to Katello.